lib/validation/validator.js

```diff
@@ -1,8 +1,36 @@
 import {fileURLToPath} from "node:url";
 import {readFile} from "node:fs/promises";
 
+/**
+ * @module @ui5/project/validation/validator
+ * @description A collection of validation related APIs
+ * @public
+ */
+
+/**
+ * @enum {string}
+ * @private
+ * @readonly
+ */
+const SCHEMA_VARIANTS = {
+	"ui5": "ui5.json",
+	"ui5.json": "ui5.json",
+	"ui5-workspace": "ui5-workspace.json",
+	"ui5-workspace.json": "ui5-workspace.json",
```

Comment:
Why are there two different keys per value?

Reply:
It seems like it's more convenient if the end user could be able to provide both variants- with the extension and without it. 
Of course, we might want to limit this to either of those